src/gui/AboutDialog.cpp

```diff
@@ -46,9 +46,12 @@ AboutDialog::AboutDialog(QWidget* parent) :
 					arg( QT_VERSION_STR ).
 					arg( GCC_VERSION ) );
 
+	copyrightLabel->setText( copyrightLabel->text().
+					arg( LMMS_PROJECT_COPYRIGHT ) );
+
```

Comment:
Not a huge fan of this logic, `me.setValue(me.getValue() + " foo");` as multiple calls can look like `foo foo foo foo` but it's already used in the constructor and it's already used in the lines preceding, it's OK.  Offtopic, but we probably need to make a decision on if we even want these `.ui` files around as they just make the code hard to read IMO. :wink: